### ISSUE TITLE
Support custom PKGBUILD repository other than AUR

### ISSUE THREAD

**Issue by brainwo** [Jguer/yay#2454]
### Is your feature request related to a problem? Please describe.

Managing multiple PKGBUILD repository is a pain without a helper.

### Describe the solution you'd like

`yay` already pulls AUR package lists as `aur` repository. Ideally, supporting another PKGBUILD repository would do the same as AUR, but with another name and in another location, with the same method of installing PKGBUILDs from AUR using `yay`.

### Describe alternatives you've considered

Switch to `paru`, but `paru` also lacks some feature I'm looking for, I prefer `yay`.

### Additional context

How `paru` handles this:

https://github.com/Morganamilo/paru/blob/130a7c678610421a077659fbe1a70355c958d2f2/man/paru.conf.5#L388

In short, it allows you to add a new section in `paru.conf` just like in `pacman.conf`.

```conf
[mylocalpkgbuildrepo]
Path = /path/to/my/local/repo

[myremotepkgbuildrepo]
Url = https://path/to/my/remote/repo
```

**Comment by Jacoblightning** [comment]
There is a flag --aururl to specify another url.
